### ISSUE TITLE
Feature Request: Make share context path configurable

### ISSUE THREAD

**Issue by toonconvents** [Alfresco/alfresco-sdk#417]
For our extension of Share we use a different context path instead of /share. Adding the possibility to configure this context path to be different from the default without completely overriding the tomcat plugin would be very useful.

Thanks

**Comment by gravitonian** [comment]
To use a custom context path for Share add the following extra property:
```
   . . .
   <enableShare>true</enableShare>
   **<shareContextPath>/share-this</shareContextPath>**
  . . .
```

**Comment by ohej** [comment]
Fixed in 1f46a9e2ffea1efd4b17a4b5d34ee39f1d849a74
